DigitalLearningSolutions.Web/Views/Register/Summary.cshtml

```diff
@@ -70,6 +70,19 @@
           </a>
         </dd>
       </div>
+      <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Centre Email

```

Comment:
Just remembered this capitalisation should also be changed, and we should probably call the field above this "Primary email"